media/css/firefox/enterprise.scss

```diff
@@ -0,0 +1,230 @@
+        background: url('/media/img/firefox/enterprise/laptop.png') top 150px left calc(50% + 270px) no-repeat,
+                    url('/media/img/firefox/enterprise/wave.png') bottom left no-repeat,
+                    url('/media/img/firefox/enterprise/circle.png') top 120px left calc(50% - 150px) no-repeat,
+                    url('/media/img/firefox/enterprise/curve.png') top 0 right calc(50% - 400px) no-repeat,
+                    linear-gradient(90deg, #003da7, #181562);
+    }
+}
+
+
+/* -------------------------------------------------------------------------- */
+// Detail content pillars
+
+.details ul {
+    li {
+        margin-bottom: 60px;
+        list-style-type: none;
+        width: 100%;
+    }
+
+    h3 {
+        margin-bottom: 20px;
+    }
+
+    p {
+        line-height: 30px;
+    }
+
+    a:link,
+    a:visited {
+        color: $color-photon-blue;
+        text-decoration: none;
+    }
+
+    a:hover,
+    a:active,
+    a:focus {
+        color: $color-photon-blue;
+        text-decoration: underline;
+    }
+
+    @media #{$mq-desktop} {
+        li {
+            float: left;
+            width: 33%;
+            margin-bottom: 0;
+        }
+
+        h3,
+        p {
+            padding-right: 40px;
+        }
+    }
+
+    @media #{$mq-desktop-wide} {
+        h3,
+        p {
+            padding-right: 60px;
+        }
+    }
+}
+
+@supports(display: grid) {
+    .details ul {
+        @media #{$mq-desktop} {
+            display: grid;
+            grid-gap: 60px;
+            grid-template-columns: repeat(3, 1fr);
+
+            li {
+                float: none;
+                width: auto;
+            }
+
+            h3,
+            p {
+                padding-right: 0;
+            }
+        }
+
+        @media #{$mq-desktop-wide} {
+            h3,
+            p {
+                padding-right: 0;
+            }
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Bottom download CTA.
+
+.secondary-download.content-section {
+    padding: 20px 0;
+
+    .download-contain {
+        position: relative;
+        padding: 40px 0;
+
+        &:before {
+            background-color: #65e1ff;
+            background-image: linear-gradient(to left, #65e1ff 0%, #95b5ff 78%, #a2a9ff 100%);
+            content: '';
+            height: 6px;
+            left: 0;
+            position: absolute;
+            top: 0;
+            width: 100%;
+        }
+    }
+
+    h2 {
+        font-weight: normal;
+        margin-bottom: 40px;
+    }
+
+    @media #{$mq-desktop} {
+        .download-contain {
+            @include clearfix;
+            padding: 100px 0 60px;
+        }
+
+        h2 {
+            float: left;
+            text-align: left;
+            width: 66%;
+        }
+
+        .download-button-wrapper {
+            float: left;
+            text-align: left;
```

Comment:
The alignment of the link contained herein feels a bit off... Maybe right align?

Reply:
Left aligned the button with the grid box above it. Right aligned felt too far away. Let me know if it still looks off though.